### DISCUSSION TITLE
Ignore prompt for everything ?

### DISCUSSION THREAD

**Discussion by randrini** [moghtech/komodo#171] [General]
Hello,;
I might have overlooked it but how do I ignore all prompt , well at least  with actions like redeploy, restart, etc ?
Thanks !

**Comment by mbecker20** [comment]
Hey, if I understand correctly, you are looking for `KOMODO_DISABLE_CONFIRM_DIALOG` https://github.com/mbecker20/komodo/blob/1f79987c58c54def5f317b79361173deab8cf13a/compose/compose.env#L42

**Reply by randrini** [reply]
Life saver !
I'm learning docker with Komodo.
I tried Portainer like for 1 hour but it was kinda convoluted for me.
Thanks for the great projet !
